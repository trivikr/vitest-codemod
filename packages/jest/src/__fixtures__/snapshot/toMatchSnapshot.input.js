it("toMatchSnapshot", () => {
  expect("foobar".toUpperCase()).toMatchSnapshot();
});
