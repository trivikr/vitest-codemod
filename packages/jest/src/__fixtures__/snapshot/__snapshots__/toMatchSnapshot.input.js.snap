// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toMatchSnapshot object 1`] = `
{
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "John Doe",
}
`;

exports[`toMatchSnapshot string 1`] = `"FOOBAR"`;
